braindecode/models/sleep_stager.py

```diff
@@ -0,0 +1,90 @@
+import torch
+from torch import nn
+
+
+class ChambonSleepStager(nn.Module):
```

Comment:
not sure about the naming here :)

Reply:
to be honest I would put the model in the example to limit the library size / scope in the future.
we can do a caffe zoo like repo later with many EEG models including pretrained ones.

what do others think?

Reply:
I guess the current `models` directory is pretty much a zoo-like collection of EEG models (7 models so far). I'm not against moving them to a separate repo, as long as it doesn't create too much overhead. What do @robintibor @gemeinl think?

Reply:
I am not suggesting to put it another repo but in the example file. To be reconsidered when used
across files.

with all the machinery of braindecode now having 20 to 40 lines in an example to write up the module
seems an ok solution. As soon as it's in the library code changing it is harder (deprecations etc..)

Reply:
I agree with @hubertjb that models is supposed to be already like zoo-like collection of EEG models. So it depends on if the model is expected to change a lot in the near future, then ok maybe keep it in example. but if it is fairly settled I think it is nice to have it in models dir

Reply:
Similar to how torchvision has quite some models as well: https://pytorch.org/docs/stable/torchvision/models.html